initfiles/bash/etc/systemd/system/esp@instance.service.in

```diff
@@ -0,0 +1,27 @@
+[Unit]
```

Comment:
Is this related? Not sure how it is connected with Spark

Reply:
This file is needed because we now start spark along with the esp.  That's why we broke it out into it's own separate file instead of having it get built by a template.